### DISCUSSION TITLE
【第5回】2024年04月07日

### DISCUSSION THREAD

**Discussion by kagomen** [first-contributions-ja/first-contributions-ja.github.io#159] [【メンテナー必読】週報]
## 目次
> 目次はトピックを立てた人が書き加えてください
- [トピックタイトル](URL)
  - 結論
 
## 次の一週間でやること
- 

***
以下フォーマット
```
## トピック: <タイトル>


## 結論


## 関連 Issues、Discussions

```

**Comment by kazzyfrog** [comment]
## 今週やったこと
- バージョン１リリースまで：
  - [x]   ドッグフーディングにて、見つかった修正箇所などをリストアップ→修正
  - [x]   READMEにスクショの追加
  - [x]   バージョンのタグと、リリースノートを作る！
- その他
  - [x] デザインやコンセプトに関するfigma会議

---

## 次の1週間でやること

下記の流れに沿って、細かいリファクタリングなどしながら、
figma？とかで、バージョン２をどういう風にするか、どこまで作り込むか？などを決める感じですかね！！

今後の大まかな流れは下記：
- バージョン1.1.0まで：
  - リファクタリングなどの改善・修正
  - バージョン２の仕様決め：
    1. コンセプト決め：リポジトリのコンセプトやストーリー性、Contributors.jsonに入力してもらうデータを決める
    2. デザインコンペ：それを元に、個別ページ？、トップの表示（カード？）などのデザインをみんなでそれぞれ作ってみる
    3. デザインカンプ用意：それぞれのデザインの良いところを取ったり混ぜたりして、1人がベースを整えてまとめる
- その後：バージョン２の実装！→確認→リリース🚀
